applet/src/gfx.h

```diff
@@ -71,7 +71,7 @@ void green_led(int on);
 void red_led(int on);
 void lcd_background_led(int on);
 
-void print_time_hook(void);
+void print_time_hook(char *log);
```

Comment:
This particular fix matches the existing actual parameter list of print_time_hook from gfx.c.

If issue #732 is correct, then both gfx.c and gfx.h will need to change in this PR.